docs/developer-guide/views-examples.md

```diff
@@ -0,0 +1,120 @@
+---
+description: Learn about the most used custom view examples of Lite XL.
+---
+
+## Toolbar
+
+### Tutorial
+
+```lua
+-- mod-version:3 -- lite-xl 2.1
+```
+
+This is required to specify Lite XL version compatibility.
+
+The instructions that begin with `require` add necessary libraries.
```

Comment:
The code block above didn't mention any `require`, so this is pointless here. I assumed that anyone reading would know a bit of lua. It might be worth pointing to the learning lua page.